components/build-service/production/kflux-ocp-p01/webhook-config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "https://github.com": "https://smee-smee.apps.stone-prd-host1.wdlc.p1.openshiftapps.com/redhathook13",
+    "https://gitlab.com": "https://smee-smee.apps.stone-prd-host1.wdlc.p1.openshiftapps.com/redhathook13"
```

Comment:
@gbenhaim This is a copy of p02, do we need to change something here?

Reply:
yes we would need to use another channel for the webhooks.
The channel is created automatically by the client, so the only thing we need is to change the path in the url, for example `https://smee-smee.apps.stone-prd-host1.wdlc.p1.openshiftapps.com/**redhathook14**`

The same url should be included in the smee-client component.

Reply:
Done